examples/animation-aframe-logo/index.html

```diff
@@ -17,8 +17,8 @@
         <a-animation attribute="position" from="0 5 0" to="0 0 0" dur="4000"
                      easing="ease-out"></a-animation>
         <a-camera id="camera" near="1000" far="4000" position="0 880 1290" fov="2.2"
-                  rotation="-34 0 0" wasd-controls-enabled="false"
-                  look-controls-enabled="false"></a-camera>
+                  rotation="-34 0 0" position-enabled="false"
```

Comment:
position-enabled and rotation-enabled they're misleading names. Position and rotation are not really disabled. They can still be changed without controls.